plutus-core/testlib/PlutusCore/Test.hs

```diff
@@ -251,6 +254,12 @@ goldenUEvalProfile
     => String -> [a] -> TestNested
 goldenUEvalProfile name values = nestedGoldenVsDocM name $ pretty . view _2 <$> (rethrow $ runUPlcProfile values)
 
+initialSrcSpan :: FilePath -> SrcSpan
```

Comment:
Haddock! The meaning of these is unclear, and where to use them. Should they be defined next to `SrcSpan`? Why not?

Reply:
This is just for testing.